nats.go

```diff
@@ -3343,7 +3359,8 @@ func (nc *Conn) createNewRequestAndSend(subj string, hdr, data []byte) (chan *Ms
 	// Create new literal Inbox and map to a chan msg.
 	mch := make(chan *Msg, RequestChanLen)
 	respInbox := nc.newRespInbox()
-	token := respInbox[respInboxPrefixLen:]
+	token := respInbox[nc.respSubLen+1:]
```

Comment:
Seems that respSubLen is used only here (and set in initNewResp), and looks more like the token offset. So instead of doing a "+1" here and "-1" when setting it, if it was renamed to respSubToken (or something similar), we could simply use it here without arithmetic, no?